src/components/Challenges.js

```diff
@@ -1,16 +1,108 @@
-import React from "react";
-import { Col, Container, Row } from "react-bootstrap";
+import React from 'react';
+import {Col, Container, Row, Carousel} from 'react-bootstrap';
+import GreenPeas from "../assets/greenpeas";
+import OnionCorn from "../assets/OnionCorn";
+import CelebratoryBoi from "../assets/CelebratoryBoi";
+import BeanstalkBoi from "../assets/BeanstalkBoi";
+const Challenges = () => {
+  return (
+    <div>
+      <Container className="mt-5" >
+        <Row>
+          <Col>
+            <h1 id="challenges" className="font-weight-bold">Tech Challenges</h1>
 
-const Challenges = () => (
-  <Container className="mt-5">
-    <Row>
-      <Col>
-        <h1 id="challenges" className="font-weight-bold">
-          Tech Challenges
-        </h1>
-      </Col>
-    </Row>
-  </Container>
-);
+          </Col>
+        </Row>
+      </Container>
+      <Carousel interval={null} touch={true}>
+        <Carousel.Item>
```

Comment:
nit: would it be possible to move all the `Carousel.Item`s to their own data file or its own variable at the top and then just map over them (like in the Resource card section)?

Reply:
Hmmm I agree, but I think with the amount of time it would take (at least for me) to do (because a couple of sections have custom CSS and links in the text), I might prefer to leave that as an "if we have time" ticket that can be worked on after we get this out there.